sw/device/tests/flash_ctrl/flash_test.c

```diff
@@ -145,6 +146,8 @@ int main(int argc, char **argv) {
   uart_init(kUartBaudrate);
   flash_init_block();
 
+  LOG_INFO("flash test!");
```

Comment:
@sriyerg 
is it okay if i merge this for now? It's not a complete fix, but just gets DV building with this test again.

Reply:
sure.